ansible_bender/cli.py

```diff
@@ -347,6 +355,20 @@ def _push(self):
         # force = self.args.force
         self.app.push(target, build_id=build_id, force=False)
 
+    def _clean(self):
+        builds = self.app.list_builds()
+        print("Cleaning images from database which are no longer present on the disk...")
+        for b in builds:
+            try:
+                run_cmd(["podman", "inspect", b.target_image],
+                        return_output=False, log_output=False)
+            except subprocess.CalledProcessError:
+                self.app.remove_build(b.build_id)
+                print(f"Build entry with ID {b.build_id} has been removed from DB as it no longer has it's corresponding image")
+                continue
+            finally:
+                print("Done!")
```

Comment:
This will print "Done" for every build in the DB - let's do "Done" after the for-loop.